tests/test_queues.py

```diff
@@ -17,6 +17,8 @@ class SimpleAsync(unittest.TestCase):
     Test sqs dispatch but with local runner
     """
     def setUp(self):
```

Comment:
We do have some inconsistent formatting now and needs to be fixed.
Can the purpose of this PR, can you just review the changes?